pacai/student/multiAgents.py

```diff
@@ -165,24 +157,22 @@ def betterEvaluationFunction(currentGameState):
 class ContestAgent(MultiAgentSearchAgent):
     """
     Your agent for the mini-contest
-    """
 
-    def __init__(self, index):
-        super().__init__(index)
-
-    def getAction(self, gameState):
-        """
-        Returns an action.
+    Method to Implement:
+    - def getAction(self, gameState):
+        - Returns an action.
```

Comment:
Same here, remove dashes. Also, make sure to be consistent with periods (i know it was in before but the others don't have them you should change this)